app/controllers/cloud_object_store_container_controller.rb

```diff
@@ -62,7 +62,7 @@ def show
   private
 
   def get_session_data
```

Comment:
@martinpovolny I didn't add this controller or even this method as part of this PR.  I only changed the string value of the title.